src/components/Search.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import "../styles/Search.css";
+import { ReactComponent as SearchIcon } from "../assets/searchIcon.svg";
+import { getImages } from "../requests/getImages";
+
+export default function Search({ setSearchResults }) {
+  const [value, setValue] = useState("");
+
+  const changeValue = (event) => setValue(event.target.value);
+
+  const handleSearch = async () => {
+    const imageResult = await getImages(value);
+    return setSearchResults(imageResult);
+  };
```

Comment:
A slight change to this function here, and you can call this when enter is pressed and tick something off your features list:
```
  const handleSearch = async (event) => {
    event.preventDefault();
    const imageResult = await getImages(value);
    return setSearchResults(imageResult);
  };
```

Reply:
Thanks Stu, I spent ages trying to figure this one out